templates/institutional-landing/reports/a11y-1.json

```
@@ -6,7 +6,7 @@
   ],
   "earl:subject": {
     "@id": "_:subject",
-    "dct:references": "https://wet-boew.github.io/GCWeb/templates/institutional-landing/demoted/institution-landing-en.html",
+    "dct:references": "https://wet-boew.github.io/GCWeb/templates/institutional-landing/deprecated/institution-landing-en.html",
```

Don't update the link in the accessibility report. They are not designed to be maintainable content but more to reflect the state at that specific moment of time of when the report was created.

```suggestion
    "dct:references": "https://wet-boew.github.io/GCWeb/templates/institutional-landing/demoted/institution-landing-en.html",
```